images/events/chadev.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Highly recommend running this (really all images for that matter) through something like https://imageoptim.com/ to remove unnecessary cruft such as this comment 😎

Reply:
Didn't even think of things like this in SVGs. 👍